15_mini_project_work/3_magic_ball_game.py

```diff
@@ -8,11 +8,30 @@
 USER_NAME_PROMPT = 'Введи свое имя:\n'
 EXCLAMATION_SIGN = '!'
 USER_NAME_GREETING = 'Добро пожаловать, '
+CONTINUE_GAME_PROMPT = 'Хочешь задать еще один вопрос?\nНажми "1", затем ENTER,' \
+                       ' если ДА\nНажми любую клавишу, затем ENTER, если НЕТ\n'
+ENTER_QUESTION_PROMPT = 'Введи свой вопрос:\n'
+YES_RESPONSE = '1'
```

Comment:
Let's make is user-friendly:
COntinue game on pressing ANY button.
Exit - if there is explicit N button press
So, user have the variety of options to continue (not just "1")

Reply:
Got it. I will do it.